### PR TITLE
rename BannedElement --> ElementProblem because we're adding problems…

### PR DESCRIPTION
**Pull request by elharo**
… for elements that are not banned such as a `<runtime>java7</runtime>` element

### CODE REVIEW

File: gcloud-eclipse-tools.launch [hunk 1]
```diff
@@ -19,7 +19,7 @@
 </listAttribute>
 <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_START_ON_FIRST_THREAD" value="true"/>
 <stringAttribute key="org.eclipse.jdt.launching.JRE_CONTAINER" value="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="-os ${target.os} -ws ${target.ws} -arch ${target.arch} -nl ${target.nl} -consoleLog"/>
+<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="${oauth_id}-os ${target.os} -ws ${target.ws} -arch ${target.arch} -nl ${target.nl} -consoleLog"/>
```
Comment (chanseokoh) [inline gcloud-eclipse-tools.launch]: Need to revert this.

Comment (elharo) [reply gcloud-eclipse-tools.launch]: This is weird. Following the instructions to set the launch configuration added this.  I wonder if something has changed in where Eclipse stores its launch config info?

Comment (chanseokoh) [reply gcloud-eclipse-tools.launch]: I bet it was just a mistake. The `${oauth_id}` should have been added to VM_ARGUMENTS, not PROGRAM_ARGUMENTS.

File: plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/XmlValidationHelper.java [hunk 2]
```diff
@@ -23,9 +23,9 @@
 interface XmlValidationHelper {
 
   /**
-   * Searches the {@link Document} for banned elements and returns them
-   * as a list of {@link BannedElement}s.
+   * Searches the {@link Document} for element problems and returns them
+   * as a list of {@link ElementProblem}s.
    */
-  ArrayList<BannedElement> checkForElements(IResource resource, Document document);
+  ArrayList<ElementProblem> checkForElements(IResource resource, Document document);
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/XmlValidationHelper.java]: Maybe this should be renamed to `checkForProblems`?

Comment (elharo) [reply plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/XmlValidationHelper.java]: done

**Pull request merged**